config/kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2019-2023 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+namePrefix: k8s-image-puller-operator-
```

Comment:
I've set the prefix to `k8s-image-puller-operator-` instead of `kubernetes-image-puller-operator-` because the longer prefix led to resource names that were too long (IIRC, it was one of the rolebindings)